karma-edit-story.config.cjs

```diff
@@ -78,9 +84,11 @@ module.exports = function (config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: config.coverage
-      ? ['progress', 'coverage-istanbul']
-      : ['progress'],
+    reporters: [
+      'progress',
+      'cuj',
```

Comment:
Should we also control it by flag? Otherwise, in the debug/watch mode it'll be really spammy. We could use the same `config.coverage` flag.